integration-cli/docker_utils.go

```diff
@@ -269,6 +269,14 @@ func (d *Daemon) Cmd(name string, arg ...string) (string, error) {
 	return string(b), err
 }
 
+func (d *Daemon) CmdWithArgs(daemonArgs []string, name string, arg ...string) (string, error) {
```

Comment:
Issue is that Cmd() does not address flags to the docker binary and also assumes sock connection. I believe this is lot simpler than overloading existing Cmd. Changing signature of Cmd will change more code in a weird way?